File: c#/state-template/src/dominio/Contexto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace state_template.src.dominio {
    using estados;
    
    class Contexto 
    {
        private EstadoContexto estado;

        public void actuar()
        {
            this.estado.actuar(this);
        }

        public void cambiarEstado(EstadoContexto estado)
        {
            this.estado = estado;
        }
    }
}

File: python/src/contexto.py
from src.estados.estado_concreto_a import EstadoConcretoA


class Contexto(object):
    __estado = None

    def __init__(self):
        self.__estado = EstadoConcretoA()

    def actuar(self):
        self.__estado.actuar(contexto=self)

    def cambiar_estado(self, estado):
        self.__estado = estado
